draft-thomson-httpbis-alt-svcb.md

```diff
@@ -57,65 +58,80 @@ informative:
 
 --- abstract
 
-HTTP alternative services
+HTTP servers deployments that include multiple service endpoints can use
+alternative services to direct clients to use a different service endpoint.
 
 This document obsoletes RFC 7838.
 
 --- middle
 
 # Introduction
 
-HTTP alternative services provide an HTTP server with a means to direct requests
-from clients to an alternative server instance.  Clients that learn about an
-alternative service can establish a connection to the identified instance, which
-- if successfully established and authenticated - can be used for future
-requests.  This design allows for nearly seamless service continuity in some
-conditions.
+HTTP origins are often comprised of multiple service endpoints.  This can be
+driven by multiple requirements, such as a need to scale by adding multiple
+physical servers, the need to place endpoints in network locations that are
+closer to clients for performance reasons, or the need to support multiple HTTP
+versions, like HTTP/2 {{H2}} or HTTP/3 {{H3}}.
 
-The use cases that might motivate a server to direct future requests to a
-different server instance vary.
+For servers that operate multiple service endpoints, it can be advantageous - or
+even necessary - to have clients make requests to a specific service endpoint.
```

Comment:
```suggestion
For servers that operate multiple service endpoints, it can be advantageous to
have clients make requests to a specific service endpoint.
```